libr/debug/p/native/w32.c

```diff
@@ -504,24 +509,12 @@ static inline int CheckValidPE(unsigned char * PeHeader) {
 	return 0;
 }
 
-static HANDLE tid2handler(int pid, int tid) {
```

Comment:
Renamed to w32_open_thread.

Also removed the looping through *all* threads each time we need an thread HANDLE for a specific thread. I replaced it by a simple OpenThread() call. 

I guess the old implementation was for queriying a thread by pid&tid in case there where duplicate tids for different processes. However (afaik) thread IDs are unique across the whole system, so no need to check for ``te32.th32OwnerProcessID == pid`` and the loop.

This seems to speed up debugging a fair bit as well it seems. (e.g. thread handles are queried each time registers are read in debugging).